### PR TITLE
fix(Yabancıdizi): change domain

### PR DESCRIPTION
**Pull request by n1xfys**
url fix

Signed-off-by: n1xfys <118199008+n1xfys@users.noreply.github.com>

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/Y/yabancıdizi/metadata.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
 		"ga_IE": "Tá yabancıdizi ar cheann de na soláthraithe seó teilifíse eachtrach Tuircis is mó ar an idirlíon.",
 		"nl": "yabancıdizi is een van de grootste Turkse buitenlandse tv-showproviders op internet."
 	},
-	"url": "yabancidizi.pro",
+	"url": "yabancidizi.co",
 	"version": "1.1.19",
```
Comment (DarkVillager) [inline websites/Y/yabancıdizi/metadata.json:16]: Bump version
```suggestion
	"version": "1.1.20",
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Comment by DarkVillager** [comment]
also title should be:
**fix(Yabancıdizi): change domain**

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of the presence working

File: websites/Y/yabancıdizi/metadata.json (hunk 1 above)
Comment (EncryptedDev) [reply websites/Y/yabancıdizi/metadata.json:16]: Should be breaking changes if the domain was changed

**Pull request closed without merging**